elasticsearch/utils/es_seed_acl

```diff
@@ -2,6 +2,8 @@
 
 source "logging"
 
+ES_HOST=${ES_HOST:-"localhost"}
+ES_PORT=${ES_PORT:-9300}
```

Comment:
sg uses transport port?  If so, that may be a problem as ES_PORT is usually the http port

Reply:
updating to be 'ES_CLUSTER_**' but should be https://github.com/openshift/origin-aggregated-logging/pull/1266#discussion_r206537804 in future